packages/api/server.js

```diff
@@ -81,13 +85,12 @@ app.get('/api/callback', async (req, res) => {
 
   const performed = await repo.performTicketIfAllowed(ticket, userPermission);
 
-  // todo send a message to the frontend to put a banner on the page indicating whether the action has been performed
   if (performed) {
     console.log('Successfully performed the action');
-    res.status(200).redirect(redirect);
+    res.status(200).redirect(redirect + ';done=' + performed);
```

Comment:
what values can performed take on?

Reply:
It's a boolean, returned by repo.performTicketIfAllowed. It will only be true if the test has been deleted.